com.hunterwittenborn.Celeste.yml

```diff
@@ -0,0 +1,56 @@
+app-id: com.hunterwittenborn.Celeste
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: celeste
+finish-args:
+  # We need to display on Wayland/X11.
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  # Users may want to sync files from anywhere on their system.
+  - '--filesystem=host'
+  # We need network access in order to sync files to remotes.
+  - '--share=network'
+  # Used for X11 performance as described at https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#f1.
+  - '--share=ipc'
+  # DBus connections we use.
+  - '--talk-name=com.hunterwittenborn.Celeste'
+  - '--talk-name=com.hunterwittenborn.Celeste.App'
+  - '--talk-name=com.hunterwittenborn.Celeste.Tray'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--talk-name=com.canonical.AppMenu.Registrar'
+  - '--talk-name=com.canonical.indicator.application'
+  # Extra libraries that we've bundled with the Flatpak in order to run.
+  - '--env=LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu:/snap/gnome-3-38-2004/119/usr/lib/x86_64-linux-gnu'
```

Comment:
I doubt this has any effect since there is no `/snap`

Reply:
If I run `flatpak run --command=sh com.hunterwittenborn.Celeste` there appears to be such. I found the needed libraries by running `find /` inside the Flatpak and then `grep`ing for the needed shared libraries, and that's where they popped up at.

Reply:
And if you run `flatpak run --nofilesystem=host --command=sh com.hunterwittenborn.Celeste`  ?

Because you already gave broad filessystem permission so the `/snap` from your system is visible.

Reply:
I can confirm that's the case. I've got a fix submitted in the upstream project that's getting deployed right now, but due to some changes in the Snap I'll probably need to wait a few days for it to get published, and then I'll get those incorporated into here.

Reply:
so what if you don't set it? It should work without because if it doesn't then it's a bug.

Reply:
There's some libraries bundled with the Snap that are needed for the application to run, and those need to also be bundled with the Flatpak, but that environment variable needs to be set so it can find the right libraries in `/app` (they'll be in there after I push some changes in a second).